### DISCUSSION TITLE
Why wont my configuration get applied after changing the "hyperland.conf" file.

### DISCUSSION THREAD

**Discussion by Kellerino** [prasanthrangan/hyprdots#41] [Q&A]
Hello freind,

first: my respect for your hard work to provide this repo for us!

My question is, if i change a shotcut for starting certain apps or want to change my monitor settings, the change wont be adapted. Is there any additional step, that has to be done in order to write the changes into my system/environment?

for example i changed this shortcut to start rofi: "bind = $mainMod, space, exec, pkill rofi || ~/.config/hypr/scripts/rofilaunch.sh d"

thank you!

**Comment by prasanthrangan** [comment]
Hey, thank you so much!

Sorry i just made significant changes to hyprland.conf to make it modular, so can you please try pulling the latest updates and reinstalling again?

**Reply by Kellerino** [reply]
Thank you for the tip. Do i have to reinstall my whole system or just hyperland?

**Reply by prasanthrangan** [reply]
Not the whole system, just pull the latest changes from my git, and execute the `./install.sh` script from your `~/clone/Scripts` directory

**Reply by Kellerino** [reply]
This helped.

After every change in my configuration files i have to run the install script in order to take the changes. Thank you.
